### PR TITLE
changes to event HTML and found zip form

### PR DESCRIPTION
**Pull request by aring97**

### CODE REVIEW

File: src/scripts/events/EventCard.js [hunk 1]
```diff
@@ -1,7 +1,78 @@
...
-    <div class="eventDate">Event date: ${eventObj.date}</div>
-    <div class="eventLocation">Event location: ${eventObj.location}</div>
+export const eventHTML=(eventObj, isFirst, isOwn)=>{
+    let monthString=""
+    switch(eventObj.date.getMonth()){
+        case 0:
+            monthString="January"
+            break
+        case 1:
+            monthString="Fedruary"
```
Comment (heymonicakay) [inline src/scripts/events/EventCard.js]: Small typo here - "Fedruary" should say "February"

File: src/scripts/events/EventList.js [hunk 2]
```diff
@@ -46,7 +46,10 @@ export const eventList=()=>{
         }else{
             let isFirst=true
             correctEvents.forEach(eventObj=>{
-                if(isFirst){
+                if(isFirst&&eventObj.userId===parseInt(sessionStorage.getItem("activeUser"))){
+                    contentTarget.innerHTML+=eventHTML(eventObj, isFirst, true)
```
Comment (mgjeffries) [inline src/scripts/events/EventList.js:53]: I see that we are passing three arguments into eventHTML, but eventHTML only takes one argument. I'd like to see eventHTML updated to have the same number of arguements.

File: src/scripts/events/EventCard.js [hunk 3]
```diff
@@ -1,7 +1,78 @@
...
+            monthString="December"
+            break
+    }
+    
+    
+    if(isFirst&&isOwn){
+        return `
+        <section class="event--first">
+    <div class="event--name--first">Event name: ${eventObj.name}</div>
+    <div class="event--date--first">Event date: ${monthString} ${eventObj.date.getDate()} ${eventObj.date.getFullYear()}</div>
```
Comment (heymonicakay) [inline src/scripts/events/EventCard.js]: Could you put a comma between the date and the year? So it outputs something like `September 2, 2020`. Right now it is outputting `September 2 2020`

File: src/scripts/events/EventForm.js [hunk 4]
```diff
@@ -21,12 +21,10 @@ eventHub.addEventListener("click", event=>{
...
-    <p>Date of  event:</p>
-    <input type="date" class="event--form__input" id="eventDate">
-    <p>Location of event:</p>
-    <input type="text" class="event--form__input" id="eventLocation">
+    <p>Name of event: <input type="text" class="event--form__input" id="eventName"></p>
+    <p>Date of  event: <input type="date" class="event--form__input" id="eventDate"></p>
+    <p>Location of event: <input type="text" class="event--form__input" id="eventLocation"></p>
+    <p>Event zipcode: <input type="text" class="event--form__input" id="eventZip"></p>
     <button id="event--form__close" class="close">Close event form</button>
     <button id="create--event">submit</button>
```
Comment (heymonicakay) [inline src/scripts/events/EventForm.js]: Could you capitalize the 's' in 'submit'?

**Review by heymonicakay — CHANGES_REQUESTED** [review]
Looks good! Just a couple edits.

**Review by heymonicakay — APPROVED** [review]
Looks great!

**Review by mgjeffries — APPROVED** [review]

**Pull request merged**